Ingredients/Startup.cs

```diff
@@ -19,7 +19,12 @@ public Startup(IConfigurationRoot configuration)
     public void ConfigureServices(IServiceCollection services, IWebHostEnvironment environment)
     {
         var neo4JSettings = new Neo4JOptions();
-        Configuration.GetSection("Neo4JOptions").Bind(neo4JSettings);
+        
+       
+         if (environment.IsProduction())
+            neo4JSettings.Neo4JConnection = new Uri("bolt://neo4j:7687");
+
+      //  Configuration.GetSection("Neo4JOptions").Bind(neo4JSettings);
```

Comment:
This is what bugs me about this PR. **It breaks the entire project.** If this made it onto dev no other team could properly run or access the program and you didn't discover it before you opened a PR. **How is that possible?** The application literally has no way of finding the database, with the line 27 added it works. 

It's hard to overstate how big of a mistake this is. You did not bother to test the application at all or, potentially worse, you didn't bother finding the reason why you were getting 500 HTTP Codes. How do you develop an application without actually running it? I find it hard to describe this as anything else than utter incompetency and failure on all levels. 

Surprisingly the rest of the code seems fine, though I haven't thoroughly tested it, and yet there is this.

Reply:
that is the thing i know that this line should be in the project wrote with flo about it; i know it should be in it but removing it was for me the only way i could use it with the docker else it always routed to localhost address only not the neo4j when i tried to implement the csv files to be in the docker integrated; and i did test it i did not get any 500 HTTP Codes i could still connect to swagger and test out getting an ingredient from the database; i did test it and even sent flo sc for how it worked when i tested it